src/llmcompressor/entrypoints/oneshot.py

```diff
@@ -228,6 +232,7 @@ def oneshot(
     preprocessing_num_workers: Optional[int] = None,
     min_tokens_per_module: Optional[float] = None,
     trust_remote_code_data: bool = False,
+    calibrate_moe_context: bool = False,
```

Comment:
Should this always be true if it's a MoE model passed into oneshot? Wondering if this can be inferred instead of requiring user setting

Reply:
We can start inferring this once this pathway has been hardened but for now I'd like to make it easy to turn off/on